### ISSUE TITLE
Looking for Maintainers

### ISSUE THREAD

**Issue by AjayP13** [plasticityai/magnitude#63]
We still plan to use and keep Magnitude alive at Plasticity, however, we are a small startup (two people as of now) and don't have time to maintain this as a full-time operation. We will eventually get to responding to some of the issues and adding new features when we are able to, but if you use this project day-to-day for your own use and have time to help maintain it, we would appreciate it. It would almost certainly be better than rolling your own solution at this point and would benefit the community of users who use this package. :) We also already have a good CI/CD infrastructure setup for it, so releasing new versions and adding tests should be easy.

Also, this is not the correct repository to send Pull Requests to. This GitHub repository is a mirror of the GitLab repository, [see here for more information](https://github.com/plasticityai/magnitude#contributing). This is another reason why we cannot just easily merge GitHub PRs here. Please send PRs to GitLab.

Please e-mail me at ajay@plasticityai.com to get setup as a contributor.

**Comment by ParikhKadam** [comment]
@AjayP13 I have already made a PR here fixing a minor but important bug. Should I do it on GitLab too?

I ain't using this project I my daily life. But yes, I did use it before. And the main thing I love about this project is it's API which is very easy-to-use. I would like helping in maintainance of this project. I get a few hours of free time every week which I can utilize here.

Thank you..

**Comment by AjayP13** [comment]
@ParikhKadam Yes, please. Could you make close the PR here and make it on GitLab. I will review it there and I can get it merged in! Thanks, that's great to hear. We made the project because we felt like an easier tool needed to exist. AFAIK, it still works right now for its core use case, but could use some bug fixes and feature upgrades.

**Comment by ParikhKadam** [comment]
@AjayP13 I haven't used gitlab much. I saw your repo on gitlab and sent an email for merge request but that was rejected due to some issues. How can I send a merge request to you on gitlab?

**Comment by AjayP13** [comment]
@ParikhKadam You should be able to go here https://gitlab.com/Plasticity/magnitude/merge_requests and hit 'New Merge Request'. Then, you should see a screen like this that lets you pick what source repository / branch to merge from.

![image](https://user-images.githubusercontent.com/5404177/68044677-08d4c800-fc95-11e9-9045-b9e8907d4aa9.png)

**Comment by ParikhKadam** [comment]
![Screenshot from 2019-11-02 13-01-38](https://user-images.githubusercontent.com/39589393/68067704-49a20f00-fd71-11e9-96bb-868deccafec1.png)
Can't find that button...

Maybe coz I haven't forked your repo on gitlab. Actually, I mirrored my patch repo from github on gitlab.

**Comment by Hellisotherpeople** [comment]
It's really sad that no one else is working on this library (I'd love to but can't for several reasons). 

as far as I can tell, it's the fastest and most effective library for using word vectors to this day. Yet, it hasn't had an update for nearly TWO years. 

This library is AWESOME and someone should continue development on it - especially with trying to get the latest GPU based models supported.

**Comment by c-chaitanya** [comment]
Hi, I have used the library before, and occasionally do use it these days too. It would be sad for this repo to go unmaintained, I would love to be a maintainer and keep this repo active.

**Comment by davebulaval** [comment]
@AjayP13 I'm interested in maintaining the project.

**Comment by ParikhKadam** [comment]
Is this actually being maintained here or on gitlab? It's too confusing when the code lies at two different places.

**Comment by c-chaitanya** [comment]
Hi @ParikhKadam, The main repository is actually maintained on Gitlab, just that the repo is available on both Github and Gitlab.

**Comment by ParikhKadam** [comment]
@c-chaitanya Yes, but then I think the issue tracking and all should also be done on Gitlab itself. This becomes too confusing when you open an issue here, fix it and do a PR here, but the actual code lies in other repo.
